### PR TITLE
Feature/ipehr 137 execute stored aql

### PR DESCRIPTION
**Pull request by SealTV**

### CODE REVIEW

**Review by Setest — APPROVED** [review]

File: src/pkg/api/query.go [hunk 1]
```diff
@@ -79,3 +84,105 @@ func (h QueryHandler) ExecPost(c *gin.Context) {
...
+
+			if fetch < 0 {
+				c.JSON(http.StatusBadRequest, gin.H{"error": "fetch cannot be less than 0"})
+				return
+			}
+
+			req.Fetch = fetch
+
+			continue
+		}
```
Comment (1db1) [inline src/pkg/api/query.go:186]: complicated, you can do it like this
```
	req.QueryParameters["ehr_id"], err = uuid.Parse(c.Query("ehr_id"))
	if err != nil {
		log.Printf("cannot parse ehr_id uuid: %f", err)
		c.JSON(http.StatusBadRequest, gin.H{"error": "ehr_id bad format"})
		return
	}


	req.Offset, err = strconv.Atoi(c.Query("offset"))
	if err != nil {
		log.Printf("cannot parse 'offset' from string: %f", err)
		c.JSON(http.StatusBadRequest, gin.H{"error": "offset bad format"})
		return
	}

	if req.Offset.(int) < 0 {
		c.JSON(http.StatusBadRequest, gin.H{"error": "offset cannot be less than 0"})
		return
	}

	req.Fetch, err = strconv.Atoi(c.Query("fetch"))
	if err != nil {
		log.Printf("cannot parse 'fetch' from string: %f", err)
		c.JSON(http.StatusBadRequest, gin.H{"error": "fetch bad format"})
		return
	}

	if req.Fetch.(int) < 0 {
		c.JSON(http.StatusBadRequest, gin.H{"error": "fetch cannot be less than 0"})
		return
	}
```

**Review by 1db1 — APPROVED** [review]

File: src/pkg/api/query.go (hunk 1 above)
Comment (SealTV) [reply src/pkg/api/query.go:186]: we can't do as you suggested. In http query we can receive varied args which can be used inside aql query

Comment (1db1) [reply src/pkg/api/query.go:186]: indeed, then we need to make a correct swagger description

Comment (SealTV) [reply src/pkg/api/query.go:186]: done

**Pull request merged**